src/mme/metrics.c

```diff
@@ -0,0 +1,83 @@
+#include "ogs-app.h"
+#include "mme-context.h"
+
+#include "metrics.h"
+
+typedef struct mme_metrics_spec_def_s {
+    unsigned int type;
+    const char *name;
+    const char *description;
+    int initial_val;
+    unsigned int num_labels;
+    const char **labels;
+} mme_metrics_spec_def_t;
+
+ogs_metrics_spec_t *mme_metrics_spec_global[_MME_METR_GLOB_MAX];
+ogs_metrics_inst_t *mme_metrics_inst_global[_MME_METR_GLOB_MAX];
+mme_metrics_spec_def_t mme_metrics_spec_def_global[_MME_METR_GLOB_MAX] = {
+[MME_METR_GLOB_GAUGE_ENB_UE] = {
+    .type = OGS_METRICS_METRIC_TYPE_GAUGE,
+    .name = "enb_ue",
+    .description = "Number of UEs connected to eNodeBs",
+    .initial_val = 0,
```

Comment:
Drop all members initialized to 0/NULL, that's already done by default and takes a lot of space (now we have 3 counters, but there may be dozens later...)